### DISCUSSION TITLE
Icon List

### DISCUSSION THREAD

**Discussion by joserochabh** [h2oai/wave#1020] [Q&A] [answered]
Hi,
Someone could me let to know how can I get all icons name available at H2o Wave?
Thanks a lot.
J. Rocha

**Comment by psinger** [comment, marked as answer]
@joserochabh 
This list should contain all:
https://uifabricicons.azurewebsites.net/

**Reply by joserochabh** [reply]
It's perfect,
Thanks a lot.
J. Roch
